index.html

```diff
@@ -5,44 +5,129 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="css/style.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
 </head>
 <body>
    <header>
+       <div class="upper_navbar">
+           <ul>
+            <li><a href="#"><img src="images/linkedinlogo.png" alt="Linked in nav item"></a></li>
+            <li><a href="#"><img src="images/twitterlogo.png" alt="Linked in nav item"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>          
+         </ul>
+       </div>
        <nav>
-           <button type="button"><img src="images/hamburger.png" alt="Hamburger button"></button>
-       </nav>
+           <button type="button" id="hamburger-btn"><img src="images/hamburger.png" alt="Hamburger button"></button>
+           <div class="desktop-navbar">
+                <a class="logo_link" href="#"><img src="images/Screenshot from 2022-02-16 15-13-24.png" alt="Homifyers logo link"></a>
+                <ul>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#categories">Categories</a></li>
+                    <li><a href="#tstmons">Top Customers</a></li>
+                    <li><a href="#brands">Brands</a></li>
+                    <li><a href="#wrhs">Visit</a></li>
+                </ul>
+            </div>
```

Comment:
- [ ] [OPTIONAL] The nav bar has too much height kindly consider reducing the size of the logo so your nav bar is thinner

![Screenshot_20221028_223319](https://user-images.githubusercontent.com/25227022/198736794-d5c0838c-1f6f-47b1-9ea9-2131af522c14.png)

Reply:
This is also the issue live version only and not locally.